app/models/concerns/roleable.rb

```diff
@@ -20,10 +20,9 @@ def regular?
 
   def whisperer?
     @whisperer ||= begin
-      return false if !SiteSetting.enable_whispers?
-      return true if staff?
       whispers_allowed_group_ids = SiteSetting.whispers_allowed_group_ids
       return false if whispers_allowed_group_ids.blank?
+      return true if admin
```

Comment:
Any reasons we're now bypassing for `admin` instead of `staff`? 🤔 

```suggestion
      return true if staff?
```

Reply:
There was an internal discussion about it (see `/t/86477/14`).